pkg/antctl/antctl.go

```diff
@@ -317,6 +318,35 @@ var CommandList = &commandList{
 			commandGroup:        flat,
 			transformedResponse: reflect.TypeOf(ovstracing.Response{}),
 		},
+		{
+			use:     "endpoint",
+			aliases: []string{"endpoints"},
+			short:   "Filter network policies relevant to an endpoint.",
+			long:    "Filter network policies relevant to an endpoint into three categories: network policies which apply to the endpoint and policies which select the endpoint in an ingress and/or egress rule.",
+			example: `  Query network policies given pod and namespace
+  $ antctl query endpoint -p pod1 -n ns1
+`,
+			commandGroup: query,
+			controllerEndpoint: &endpoint{
+				nonResourceEndpoint: &nonResourceEndpoint{
+					path: "/endpoint",
+					params: []flagInfo{
+						{
+							name:      "namespace",
+							usage:     "Namespace of the entity (required)",
```

Comment:
nit: we could easily support namespace=default if no namespace provided?

Reply:
good idea, I'll make the changes